python-modules/cffi-1.5.2/setup.py

```diff
@@ -1,6 +1,7 @@
 import sys, os
 import subprocess
 import errno
+import _ssl
```

Comment:
I think that we need a comment to describe why we are doing this awkward import.

but with the current build. do we need it?